File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Java/Exercises/21_compito_java_2018_04_06a/Menu.java
public class Menu {
	protected int codiceRistorante;
	protected String descrizioneProdotto;
	protected String tipoProdotto;
	protected float prezzoProdotto;
	protected float costoMateriePrime;

	public Menu(int codiceRistorante, String descrizioneProdotto, String tipoProdotto, float prezzoProdotto,
			float costoMateriePrime) {
		this.codiceRistorante = codiceRistorante;
		this.descrizioneProdotto = descrizioneProdotto;
		this.tipoProdotto = tipoProdotto;
		this.prezzoProdotto = prezzoProdotto;
		this.costoMateriePrime = costoMateriePrime;
	}

	public String toString() {
		return descrizioneProdotto + ": " + prezzoProdotto + "-" + costoMateriePrime;
	}
}

File: Programming-languages-and-laboratory/Python/Exercises/21a_compito_java_20180406/Menu.py
class Menu:
    def __init__(self, codice, descrizione_prodotto, tipo, prezzo, costo_materie_prime):
        self._codice = codice
        self._descrizione_prodotto = descrizione_prodotto
        self._tipo = tipo
        self._prezzo = prezzo
        self._costo_materie_prime = costo_materie_prime

    def __str__(self):
        return self._descrizione_prodotto + ": " + str(self._prezzo) + "-" + str(self._costo_materie_prime)
